priv/repo/migrations/20160805203929_create_skill.exs

```diff
@@ -0,0 +1,16 @@
+defmodule CodeCorps.Repo.Migrations.CreateSkill do
+  use Ecto.Migration
+
+  def change do
+    create table(:skills) do
+      add :title, :string
+      add :description, :string
+      add :original_row, :integer
+      add :slug, :string
+
+      timestamps()
+    end
+
+    create index(:skills, [:slug], name: :index_skills_on_slug, unique: true)
```

Comment:
The idea is to differentiate between `code-corps` and `CODE-CORPS` which should resolve to the same thing. I don't believe uniqueness will deal with these constraints.